src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export class TodoForm extends React.Component {
+    state = {
+        task: ''
+    }
+
+    onSubmit = (e) => {
+        e.preventDefault();
+        // check if it is user input present
+        if(this.state.task !== "") {
```

Comment:
this is my favorite part so far, great job covering this edge case, it is often forgotten but it's the first thing a user notices when they use your apps.